scattering/van_hove.py

```diff
@@ -121,6 +123,10 @@ def compute_van_hove(
                     )
                 )
 
+            if elem1.symbol > elem2.symbol:
```

Comment:
This looks good.  Could you also add to the docstring that we are ordering the pairs in alphabetical order?